src/MacroTools/Cheats/TestMode.cs

```diff
@@ -21,7 +21,7 @@ public static class TestMode
     public static bool CheatCondition()
     {
       var triggerPlayerName = GetPlayerName(GetTriggerPlayer());
-      return triggerPlayerName is "YakaryBovine#6863" or "Lordsebas#11619" || AreCheatsActive;
+      return triggerPlayerName is "YakaryBovine#6863" or "Lordsebas#11619" or "Technopig#2179" || AreCheatsActive;
```

Comment:
You've got a conflict on this change, which makes sense because your branch from `dev` occurred before I added Vampirika to this line. To resolve it you just need to merge `dev` back into your `Technoopig/Balance` branch, then manually resolve the git conflict that pops up. Alternatively you could revert this change and I'll just do it when I get a chance!

Reply:
Ah sorry only saw this comment after I "fixed" the conflict , but I merged "Technoopig/Balance" into "main" instead as I think the conflict was there. Let me know if it needs redoing and ill just take the line out :D

Reply:
No you're actually correct, `main` was right, my bad haha

Reply:
Oops, you're right actually, you did want `main` rather than `dev`.
However take a look at the change you've got now - you're taking Vamp's name out!